compiler/rustc_mir_build/src/builder/scope.rs

```diff
@@ -180,17 +180,16 @@ struct BreakableScope<'tcx> {
 
 #[derive(Debug)]
 struct ConstContinuableScope<'tcx> {
-    /// The if-then scope or arm scope.
+    /// The scope for the `#[loop_match]` which this `#[const_continue]` will jump to.
```

Comment:
This struct is for the `#[loop_match]`, not for the `#[const_continue]`.

Reply:
sure, I've rephrased that